### PR TITLE
FIX Windows enter/exit mouse notifications

### PR DESCRIPTION
**Pull request by Ranoller**
Fix to this issue #17202 . I don´t believe that nobody notice that!

### CODE REVIEW

File: scene/main/scene_tree.cpp [hunk 1]
```diff
@@ -620,6 +620,12 @@ void SceneTree::_notification(int p_notification) {
 
 			get_root()->propagate_notification(p_notification);
 		} break;
+		case NOTIFICATION_WM_MOUSE_ENTER: {
```
Comment (hpvb) [inline scene/main/scene_tree.cpp]: These cases should probably be added to the group in line 619 to make it look like this:
```c++
                case NOTIFICATION_OS_MEMORY_WARNING:
                case NOTIFICATION_WM_FOCUS_IN:
                case NOTIFICATION_WM_FOCUS_OUT: 
                case NOTIFICATION_WM_MOUSE_ENTER: 
                case NOTIFICATION_WM_MOUSE_EXIT: {

                        get_root()->propagate_notification(p_notification);
                } break;
```
I'm not super familiar with this code but if these should be propagated like that there then that is probably how to do it!

Comment (Ranoller) [reply scene/main/scene_tree.cpp]: Oh well... I´m not familiar at all with c++ syntax :D :D :D :D....I try the fix and works but the two bots report error so i fix that.

**Comment by hpvb** [comment]
You can reuse this pull request. See our [PR workflow](http://docs.godotengine.org/en/3.0/community/contributing/pr_workflow.html) documentation how to do that.

**Comment by Ranoller** [comment]
I´m using Github web, not Git. I can send other fix-commit but @Akien don´t like that. I can´t squash commit from browser.

**Comment by akien-mga** [comment]
I'd advise you to use Git (ideally from the command line, if you must from a decent client), simply because making changes to the source code via a browser means that you can't test your own changes, so we can't trust them to do what they say.

If you did test them locally in a git clone, then it's easier to push them to a fork using Git than to redo them manually in the GitHub web editor.

**Comment by Ranoller** [comment]
I test the changes before commit, this is not related with git. You can use git and not test, you can change code and compile without use git.... I'm to bussy now but i thing in study git when i have time

**Pull request closed without merging**